From: Patrycja Rosa <alpine@ptrcnull.me>
Date: Fri, 11 Feb 2022 19:02:34 +0100
Subject: Don't install manpages with setup.py

They're not in the correct place when using the GitHub release
and from what I can see, they get installed to /usr anyway

diff --git a/setup.cfg b/setup.cfg
--- a/setup.cfg
+++ b/setup.cfg
@@ -67,6 +67,0 @@
-
-[options.data_files]
-. = scons.1
-    scons-time.1
-    sconsign.1
-
